samples/bluetooth/mesh/light_switch/prj.conf

```diff
@@ -26,6 +26,11 @@ CONFIG_BT_L2CAP_TX_BUF_COUNT=32
 CONFIG_BT_OBSERVER=y
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_SETTINGS=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=6
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_ADV_SET=6
+CONFIG_BT_MAX_CONN=2
```

Comment:
Why did you set it to 2?

Reply:
If this is not set to 2 it will give a warning when booting up the device. So I figured out that it was best to increase this to 2 so people do not wonder what the warning is about.

Reply:
The reason for this not automatically being set to 2 is that we add the OBSERVER instead of the CENTRAL. If we include the CENTRAL instead it will be set to 2, but we do not need the CENTRAL. I think that this need to be 2 because we include the OBSERVER.